bindings/gumjs/gumscript-runtime-objc.js

```diff
@@ -443,7 +486,22 @@
                                 }
                             }
                             return cachedProtocols;
+                        case "$methods":
+                            if (cachedNativeMethodNames === null) {
+                                cachedNativeMethodNames = [];
+
+                                // Fill cachedMethodNames
+                                this.keys();
+
+                                cachedNativeMethodNames = Object.keys(cachedMethods).filter(m => m.match(/^(\+|-)/));
+                            }
+                            return cachedNativeMethodNames;
                         default:
+                            if (handle.isNull()) {
+                                return function() {
+                                    return 0;
```

Comment:
The problem with this is that in Objective-C `0`, `NULL` and `nil` is the same, but there are not the same in Frida.
So if you expected the method to return an int, it would work in Objective-C, but not if we use `ObjC.nil`.
I don't know if the best choice is `0`, `NULL` or `ObjC.nil`.